src/GitHub.App/Services/ImageDownloader.cs

```diff
@@ -30,7 +30,6 @@ public IObservable<byte[]> DownloadImageBytes(Uri imageUri)
                 BaseAddress = new Uri(imageUri.GetLeftPart(UriPartial.Authority)),
                 Endpoint = new Uri(imageUri.PathAndQuery, UriKind.RelativeOrAbsolute),
                 Method = HttpMethod.Get,
-                AllowAutoRedirect = true
```

Comment:
It's deprecated and ignored due to redirect work, says the comment coming from this commit: https://github.com/octokit/octokit.net/commit/3acdd10be4800f376f5f008fd202b3a776409695. The code analysis bails on deprecation warnings, so I took all the calls out. 

I have *not* tested everything, so we might want to hold on merging this for now.